ecommerce/settings/production.py

```diff
@@ -87,3 +87,5 @@ def get_env_setting(setting):
 
 # Edunext setting: url path to the campus romero open edx extensions plugin
 OPENEDX_EXTENSIONS_API_URL = environ.get('OPENEDX_EXTENSIONS_API_URL', OPENEDX_EXTENSIONS_API_URL)
+
+PAYU_RECEIPT_URL = environ.get('PAYU_RECEIPT_URL', PAYU_RECEIPT_URL)
```

Comment:
I think that this is not needed. Ecommerce is already configured to override the settings with the values in /edx/etc/ecommerce.yml